mlir/lib/Dialect/Tensor/IR/TensorTilingInterfaceImpl.cpp

```diff
@@ -246,6 +246,103 @@ struct PackOpTiling
       return failure();
     return tilingResult.value();
   }
+
+  /// Method to return the position of iteration domain tile computed by the
+  /// tiled operation. In current `tensor.pack` context, the `resultOffsets` and
+  /// `resultSizes` only cover outer dimensions.
+  LogicalResult getIterationDomainTileFromOperandTile(
+      Operation *op, OpBuilder &b, unsigned operandNumber,
+      ArrayRef<OpFoldResult> offsets, ArrayRef<OpFoldResult> sizes,
+      SmallVectorImpl<OpFoldResult> &resultOffsets,
+      SmallVectorImpl<OpFoldResult> &resultSizes) const {
+    if (operandNumber != 0)
```

Comment:
Do you think we should early bail if the pack operation has padding semantics?

Reply:
Yeah, I am thinking so as well. That will make what is supported more clear.

Reply:
Add early bail and more comments to explain what exact case we could support at present. Thanks for advice!